client/src/components/base/blog/style.module.css

```diff
@@ -0,0 +1,79 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+.container__first__card {
+  width: 373px;
+  height: 450px;
+  margin-top: 50px;
+}
+.container__card__content span {
+  font-size: 12px;
+  display: block;
+  margin-bottom: 10px;
+  margin-top: 10px;
+}
+.container__card__content h5 {
+  font-size: 18px;
+  line-height: 1.44444;
+}
+.container__card__content h5 a {
+  color: #151515;
+  font-family: 'Oswald', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Helvetica Neue', Arial, sans-serif;
+  font-weight: 500;
+}
+.container__link:hover {
+  color: #737576;
+  cursor: pointer;
+  transition: 0.4s;
+}
+.container__card__content p {
+  margin-top: 15px;
+}
+
+.container__second__card {
+  width: 373px;
+  height: 450px;
+  margin-left: 10px;
+  margin-top: 50px;
+}
+.container__third__card {
+  width: 373px;
+  height: 450px;
+  margin-left: 10px;
+  margin-top: 50px;
+}
+.container__cards {
+  display: flex;
+  margin-top: 20px;
+  gap: 10px;
+  display: flex;
+  flex-wrap: wrap;
+}
+.container__title {
+  color: #151515;
+  font-family: 'Oswald', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Helvetica Neue', Arial, sans-serif;
+  font-size: 36px;
+  line-height: 1.11111;
+  font-weight: 500;
+}
+.container {
+  margin-left: 80px;
+  margin-right: 80px;
```

Comment:
width: 100%;
    padding-right: 15px;
    padding-left: 15px;
    margin-right: auto;
    margin-left: auto;

Take good care of your review!